src/eq_schema/schema/Block/index.test.js

```diff
@@ -15,13 +15,13 @@ describe("Block", () => {
       block
     );
 
-  const createPipedFormat = (placeholder, source) => ({
+  const createPipedFormat = (placeholder, identifier, source) => ({
     text: `{${placeholder}}`,
     placeholders: [
       {
         placeholder,
         value: {
-          identifier: placeholder,
+          identifier: identifier,
```

Comment:
```suggestion
          identifier
```

A small syntax change